docs/application/native/guides/ui/efl/efl-util.md

```diff
@@ -10,9 +10,9 @@ You can take advantage of various EFL utilities in your application:
 
   You can [capture screenshots](#taking-screenshots) and store the screenshot data as a TBM surface, using the EFL UTIL SCREENSHOT API (for [mobile](../../../api/mobile/latest/group__CAPI__EFL__UTIL__SCREENSHOT__MODULE.html) and [wearable](../../../api/wearable/latest/group__CAPI__EFL__UTIL__SCREENSHOT__MODULE.html) applications).
 
-- Using the input generator
+- Using input generator
 
-  You can generate input events using the EFL UTIL INPUT API (for [mobile](../../../api/mobile/latest/group__CAPI__EFL__UTIL__INPUT__MODULE.html) and [wearable](../../../api/wearable/latest/group__CAPI__EFL__UTIL__INPUT__MODULE.html) applications). You can generate [key input events](#generating-key-input-events) that emulate various hardware key presses and [touch input events](#generating-touch-input-events) that emulate screen touches.
+  You can generate input events using the EFL UTIL INPUT API (for [mobile](../../../api/mobile/latest/group__CAPI__EFL__UTIL__INPUT__MODULE.html) and [wearable](../../../api/wearable/latest/group__CAPI__EFL__UTIL__INPUT__MODULE.html) applications). You can generate [key input events](#generating-key-input-events) that emulate various hardware key presses, [touch input events](#generating-touch-input-events) that emulate screen touches, and [point input events](#generating-point-input-events) that emulate mouse move/buttons.
```

Comment:
It is recommended to avoid using "/". Also, "mouse clicks" is used than "mouse buttons".
Can we rephrase to:

You can generate [key input events](#generating-key-input-events) that emulate various hardware key presses, [touch input events](#generating-touch-input-events) that emulate screen touches, and [point input events](#generating-point-input-events) that emulate mouse moves or clicks.

@jhyuni and @mijin85cho 
Please verify and share your opinion.